### PR TITLE
fix(k8spsphostnetworkingports): null check container.ports

### PR DESCRIPTION
**Pull request by julianKatz**
Previously, the `badContainers` section of the K8sNative validation logic did not verify that the `ports` variable was set on a container spec in a pod.  As this template's purpose is to verify that the specified ports fall within a given range, we would expect that _no ports_ is not a violation.

### CODE REVIEW

File: website/docs/validation/host-network-ports.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ metadata:
   name: k8spsphostnetworkingports
   annotations:
     metadata.gatekeeper.sh/title: "Host Networking Ports"
-    metadata.gatekeeper.sh/version: 1.1.1
+    metadata.gatekeeper.sh/version: 1.2.0
```
Comment (apeabody) [inline website/docs/validation/host-network-ports.md:19]: As this is a Rego "fix", probably just bump the patch?

"Patch version bump required: Whenever there is a simple backward compatible change in the policy, e.g. Simple Rego fix, updating policy metadata"

```suggestion
    metadata.gatekeeper.sh/version: 1.1.2
```

**Review by maxsmythe — APPROVED** [review]
+1 to patch version bump only, otherwise LGTM

File: src/pod-security-policy/host-network-ports/constraint.tmpl [hunk 2]
```diff
@@ -4,7 +4,7 @@ metadata:
   name: k8spsphostnetworkingports
   annotations:
     metadata.gatekeeper.sh/title: "Host Networking Ports"
-    metadata.gatekeeper.sh/version: 1.1.1
+    metadata.gatekeeper.sh/version: 1.2.0
```
[the remark this answers is not in the document]
Comment (julianKatz) [reply src/pod-security-policy/host-network-ports/constraint.tmpl:7]: Updated to 1.1.2

**Review by apeabody — APPROVED** [review]
Thanks @julianKatz!

**Review by JaydipGabani — APPROVED** [review]

**Pull request merged**